packages/hop-node/src/cli/cli.ts

```diff
@@ -22,7 +22,6 @@ import './send'
 import './settle'
 import './showConfig'
 import './stake'
-import './stakeStatus'
```

Comment:
why was the stake status command removed?

Reply:
Reintroduced with adf38fdc40b6963969631da597912c2148487588